consensus/src/round_manager_test.rs

```diff
@@ -2311,19 +2333,19 @@ pub fn forking_retrieval_test() {
         3,
     );
 
-    let next_message = timed_block_on(&runtime, nodes[proposal_node].next_network_message());
-    match next_message {
-        ConsensusMsg::VoteMsg(_) => info!("Skip extra vote msg"),
-        ConsensusMsg::ProposalMsg(msg) => {
-            // put the message back in the queue.
-            // actual peer doesn't matter, it is ignored, so use self.
-            let peer = nodes[proposal_node].signer.author();
-            nodes[proposal_node]
-                .pending_network_events
-                .push(Event::Message(peer, ConsensusMsg::ProposalMsg(msg)))
-        },
-        _ => panic!("unexpected network message {:?}", next_message),
-    }
+    // let next_message = timed_block_on(&runtime, nodes[proposal_node].next_network_message());
```

Comment:
Why is this commented out? If this test is incomplete, can we add a `unimplemented!` here instead so it can break when someone tries to run it.

Reply:
Yeah. Actually, the test is flaky and there is already `#[ignore]` tag added on top of the test. The test doesn't affect forge.

Reply:
I meant if someone tries to remove the `#[ignore]` runs and thinks that the test passes because parts of code is commented out.